resources/views/gistlogs/show.blade.php

```diff
@@ -26,14 +26,14 @@
             </a>
         </div>
     @endif
-    <div class="gistlog py-8 sm:px-8">
-        <article class="my-8 px-4 sm:px-8 my-8">
+    <div class="py-8 gistlog sm:px-8">
+        <article class="px-4 my-8 sm:px-8">
             <h1 class="gistlog__title" v-pre>{{ $gistlog->title }}</h1>
-            <span class="font-light mx-auto table" v-pre>
+            <span class="table mx-auto font-light" v-pre>
```

Comment:
What Tailwind ordering system are you using here?

Reply:
Using this VSCode extension called Headwind by Ryan Heybourn: https://marketplace.visualstudio.com/items?itemName=heybourn.headwind